pkg/pillar/cmd/zedrouter/flowstats.go

```diff
@@ -89,6 +89,7 @@ const (
 	markMask        uint32 = 0xffffff // get the Mark bits for ACL number
 	appShiftBits    uint32 = 24       // top 8 bits for App Number
 	maxFlowPack     int    = 125      // approximate 320 bytes per flow/dns, got an assert in zedagent when size was 241
+	flowStaleSec    int64  = 1800     // 30 min not touched, the publication will be removed
```

Comment:
If the packet/byte count is increased, will it be considered "touched"?

Reply:
this is the flowlog part, it's not related to the MetricMap counters. each app-networkinstance creates an publication file with that key. if that file with that key is not updated by the timestame, then it's already been uploaded to the cloud and it's stale.